ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-Settle-nonprod.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
```

Comment:
Should these changes in prod files? As these changes are directly going to prod as feature (Settle claim for lips) is already live.

Reply:
moved changes to AuthorisationCaseField-R2-CUI.json